integration-tests/src/test/java/org/apache/druid/tests/coordinator/duty/ITAutoCompactionTest.java

```diff
@@ -237,7 +237,7 @@ private void verifyQuery(String queryResource) throws Exception
         fullDatasourceName
     );
 
-    queryHelper.testQueriesFromString(queryResponseTemplate, 2);
+    queryHelper.testQueriesFromString(queryResponseTemplate, 5);
```

Comment:
I don't think this will help. The 'timesToRun' number requires ALL of the running to pass. If any one of the running failed then the test will fail. Increasing timesToRun will make the test slower but does not help with retrying intermittent failure. I have a PR to fix this issue at https://github.com/apache/druid/pull/9739 
Sorry for the trouble from the intermittent failure.

Reply:
Reverting, thanks! :)